Assets/Source/Components/UI/AmmoCountBehavior.cs

```diff
@@ -23,9 +24,11 @@ public override void ComponentStart()
             GameObject ammoCount = GetRequiredChild("AmmoCount");
             ammoCountText = GetRequiredComponent<TextMeshProUGUI>(ammoCount);
 
-            actorBehavior = GetPlayerActorBehavior();
-            playerBehavior = GetPlayerBehavior();
+            GameObject player = GetRequiredObject(GameObjects.Actors.Player);
+            actorBehavior = GetRequiredComponent<ActorBehavior>(player);
+            playerBehavior = GetRequiredComponent<PlayerBehavior>(player);
 
+            ammoCountText.SetText($"x{actorBehavior.BlasterAmmo}");
```

Comment:
this needed to be set initially or else it didn't update the font size and ammo count until you shot